BasePreparedModel.cpp

```diff
@@ -64,6 +65,33 @@ static Return<void> notify(const sp<V1_2::IExecutionCallback>& callback, const E
     return callback->notify_1_2(status, outputShapes, timing);
 }
 
+static Return<void> notify(const sp<V1_3::IExecutionCallback>& callback, const ErrorStatus& status,
+                           const hidl_vec<OutputShape>& outputShapes, Timing timing) {
+    return callback->notify_1_3(convertToV1_3(status), outputShapes, timing);
+}
+
+static void floatToUint8(const float* src, uint8_t* dst, size_t size) {
+    for (uint32_t i = 0; i < size; ++i) {
+        dst[i] = static_cast<uint8_t>(src[i]);
+        ALOGV("%s input: %f output: %d ", __func__, src[i], dst[i]);
+    }
+}
```

Comment:
can this live in utils.cpp/h?

Reply:
It is part of old code and my changes.
Looks like this is not used in any other place. So may not be needed to put in utils.h .

Reply:
Okay makes sense.